src/main/java/rx/exceptions/Exceptions.java

```diff
@@ -31,7 +31,7 @@
 
     /** Utility class, no instances. */
     private Exceptions() {
-        throw new IllegalStateException("No instances!");
+        throw new AssertionError("No instances.");
```

Comment:
I find these as unnecessary changes. The methods throw to those who instantiate the class via reflection and are not test methods themselves to fail with AssertionError like `TestSubscriber.assertX`.